chargebee/src/main/java/com/chargebee/android/billingservice/BillingClientManager.kt

```diff
@@ -333,15 +451,34 @@ class BillingClientManager(context: Context) : PurchasesUpdatedListener {
 
             else -> {
                 if (purchaseProductParams.product.type == ProductType.SUBS)
-                    purchaseCallBack?.onError(
-                        throwCBException(billingResult)
-                    )
+                    if (this::changeProductParams.isInitialized && this.changeProductParams.currentProductId.isNotEmpty() && billingResult.responseCode == ERROR) {
```

Comment:
Where is `isInitialized` defined?

Reply:
It is inbuild in kotlin. Returns true if this lateinit property has been assigned a value, and false otherwise.